the-experts-BE/routes/contractors.js

```diff
@@ -0,0 +1,20 @@
+var express = require("express");
+var router = express.Router();
+var queries = require("../queries/index");
+
+/* GET contractors listing. */
+router.get("/", (req, res) => {
+  res.send("Hello World");
+});
+
+router.get("/:job", (req, res) => {
+  console.log(req.params);
+  queries.getContractor(req.params.job, (err, response) => {
+    console.log(req.params);
+    if (err) console.log(err);
+
+    res.send(response);
```

Comment:
You're sending the whole `response` object here which is unnecessary. All your FE cares about is `response.rows` so this is what you should send back.

Also, you should use `res.json()` to make it clear that you are sending back JSON to FE.